Azure/cloud-init.txt

```diff
@@ -0,0 +1,164 @@
+#!/bin/bash
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Description: This script will install all dependencies
+# required by clear containers and metrics/tests.
+#
+# OS: ubuntu 16.04
+
+# Add repository key
+wget -nv https://download.opensuse.org/repositories/home:clearcontainers:clear-containers-3/xUbuntu_16.04/Release.key -O Release.key
+apt-key add - < Release.key
+apt-get update
+
+# Add repositories and install runtime
+echo "Installing Clear Containers"
+sh -c "echo 'deb http://download.opensuse.org/repositories/home:/clearcontainers:/clear-containers-3/xUbuntu_16.04/ /' > /etc/apt/sources.list.d/cc-runtime.list"
+apt-get update
+apt-get install -y cc-runtime
+
+
+# Install docker
+echo "Installing docker"
+apt-get install -y \
+	apt-transport-https \
+	ca-certificates \
+	curl \
+	software-properties-common
+
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+apt-key fingerprint 0EBFCD88
+
+add-apt-repository \
+	"deb [arch=amd64] https://download.docker.com/linux/ubuntu \
+	$(lsb_release -cs) \
+	stable"
+
+apt-get update
+apt-get install -y docker-ce
+
+# Install tests dependencies
+apt-get install -y \
+	bc \
+	smem \
+	apache2-utils \
+	iperf
+
+# Install thin tools for devicemapper configuration
+apt-get install -y thin-provisioning-tools
+
+# Configure the clear containers runtime using devicemapper as
+# storage driver
+mkdir -p /etc/systemd/system/docker.service.d/
+cat << EOF | sudo tee /etc/systemd/system/docker.service.d/clr-containers.conf
+[Service]
+ExecStart=
+ExecStart=/usr/bin/dockerd -D --add-runtime cc-runtime=/usr/bin/cc-runtime --default-runtime=cc-runtime
+EOF
+
+# Add devicemapper configuration
+mkdir -p /etc/docker
+
+mkdir -p /etc/lvm/profile
+cat << EOF | sudo tee /etc/docker/daemon.json
+{
+"storage-driver": "devicemapper",
+"storage-opts": [
+        "dm.directlvm_device=/dev/sdc",
+        "dm.thinp_percent=95",
+        "dm.thinp_metapercent=1",
+        "dm.thinp_autoextend_threshold=80",
+        "dm.thinp_autoextend_percent=20",
+        "dm.directlvm_device_force=true"
+]
+}
+EOF
+
+# Load devicemapper changes
+systemctl daemon-reload
+
+# Update kernel
+echo deb http://archive.ubuntu.com/ubuntu/ xenial-proposed restricted main multiverse universe >> /etc/apt/sources.list
+apt-get update
+apt-get install -y linux-azure
+
```

Comment:
@MarioCarrilloA  Can we verify that this kernel update and the final reboot are necessary ? I believe they're not.
Besides this, this PR now looks fine to me.

Reply:
@MarioCarrilloA I confirm that it is not needed. I can start k8s and launch a CC based pod without having to install this kernel. Please remove that part and the reboot at the end.

Reply:
@sameo Done